ext/iframe/tests/phpunit/CRM/Iframe/IframeTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use CRM_Iframe_ExtensionUtil as E;
+use Civi\Test\EndToEndInterface;
+
+/**
+ * @group e2e
+ */
+class CRM_Iframe_IframeTest extends \PHPUnit\Framework\TestCase implements EndToEndInterface {
+
+  use \Civi\Test\HttpTestTrait;
+
+  public static function setUpBeforeClass(): void {
+    \Civi\Test::e2e()->installMe(__DIR__)->apply();
+  }
+
+  /**
+   * Enable iframe connector and send a request.
+   */
+  public function testBasicRequest(): void {
+    if (!Civi::service('iframe')->isSupported()) {
+      $this->markTestSkipped('iframe extension does not support activation in this environment');
+    }
+
+    \Civi\Api4\Iframe::installScript()->setCheckPermissions(FALSE)->execute();
```

Comment:
I may have made a small mistake in framing the guard. This setup instruction on line 25 isn't actually needed or runnable on wpmaster, so [phpunit-core-exts gives a test-failure on wpmaster.](https://test.civicrm.org/job/CiviCRM-Test/71001/testReport/junit/(root)/CRM_Iframe_IframeTest/testBasicRequest/). Working on a revision.